ext/re/t/intflags.t

```diff
@@ -0,0 +1,25 @@
+#!./perl
+
+BEGIN {
+    require Config;
+    if (($Config::Config{'extensions'} !~ /\bre\b/) ){
+	print "1..0 # Skip -- Perl configured without re module\n";
+	exit 0;
+    }
+}
+
+use strict;
+
+# must use a BEGIN or the prototypes wont be respected meaning
+    # tests could pass that shouldn't
```

Comment:
Is there some reason this is indented?

Reply:
It was indented like that in the file I cribbed it from lexical_debug.t, i fixed it in both and repushed.